morgan/__init__.py

```diff
@@ -64,6 +67,19 @@ def __init__(self, index_path: str):
 
         self._processed_pkgs = {}
 
+    @classmethod
+    def add_arguments(cls, parser: argparse.ArgumentParser):
+        """
+        Used to add mirror-specific command line options.
+        """
+
+        parser.add_argument(
+            '--mirror-config',
```

Comment:
I think a `-c` short option would be good here too, same as many other CLI apps.